pytest_django/fixtures.py

```diff
@@ -64,6 +67,13 @@ def teardown_database():
         request.addfinalizer(teardown_database)
 
 
+def is_xdist_one_db_enabled(config):
+    from django.conf import settings
+    is_sqlite = (settings.DATABASES['default']['ENGINE'] == 'django.db.backends.sqlite3')
+    # can't use one sqlite3 db for distributed test run because of lock
```

Comment:
I'm not really familiar with error handling in pytest, could you give some example or exception class name?